src/main/java/org/fundacionjala/coding/dennis/FizzBuzz.java

```diff
@@ -0,0 +1,41 @@
+package org.fundacionjala.coding.dennis;
+
+/**
+ *
+ */
+public  final class FizzBuzz {
+
+    public static final int THREE = 3;
+    public static final int FIVE = 5;
+    public static final int MCM = THREE * FIVE;
+    public static final int LIMIT = 200;
+    /**
+     *
+     */
+    private FizzBuzz() {
+    }
+
+    /**
+     *
+     * @param limit limite
+     */
+
+    public static void fBuzz(int limit) {
+        StringBuilder lString;
+        for (int pt = 1; pt <= limit; pt++) {
+            lString = (pt % MCM == 0) ? (new StringBuilder("FizzBuzz"))
+                    : ((pt % THREE == 0) ? (new StringBuilder("Fizz"))
+                    : ((pt % FIVE == 0) ? (new StringBuilder("Buzz"))
+                    : (new StringBuilder(Integer.toString(pt)))));
+            System.out.println(lString.toString());
```

Comment:
![MINOR](https://sonarsource.github.io/sonar-github/severity-minor.png 'Severity: MINOR') Replace this use of System.out or System.err by a logger. [![rule](https://sonarsource.github.io/sonar-github/rule.png)](https://sonarcloud.io/coding_rules#rule_key=squid%3AS106)